BusBoard/BusBoard/IApiHandler.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using RestSharp;
+
+namespace BusBoard
+{
+    public interface IApiHandler
+    {
+        string BaseUrl { get; set; }
+        IRestClient Client { get; set; }
+        
+        T Execute<T>(RestRequest request) where T : new();
```

Comment:
So I would probably also remove this, at which point you can also remove this interface.